units_cpp/src/units/headers/units/constants/earth_rotation.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * earth_rotation.h
+ *
+ *  Created on: Nov 3, 2017
+ *      Author: oriash93
+ */
+
+#ifndef SRC_UNITS_HEADERS_UNITS_CONSTANTS_EARTH_ROTATION_H_
+#define SRC_UNITS_HEADERS_UNITS_CONSTANTS_EARTH_ROTATION_H_
+
+#include <units/angular_speed.h>
+
+namespace units {
+namespace earth_rotation
+{
+constexpr angular_speed::DegreesPerSecond earthRotation{0.004166666666667};
```

Comment:
Why won't you save this constant is "earthSpinDuration", which is 23 hours 56 minutes and 4.1 seconds, and save other planets day length too?

That would be awesome!

Reply:
https://www.universetoday.com/37481/days-of-the-planets/